### PR TITLE
docs: rfc for researching cloud services

### PR DESCRIPTION
**Pull request by ono-max**

### CODE REVIEW

File: docs/rfcs/0039-commonalize-cloud-service-interface.md [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+| Subscription per topic             | 10,000        | 12,500,000                                                   | 2,000                                                        |
+| Expiration period in topic         | 31 days       | 4 weeks                                                      | 14 days                                                      |
+| Expiration period in subscription  | 31 days       | 14 days (default is 4 days)                                  | 14 days                                                      |
+| API throttling(Tokyo region)       |               | 1500 transactions per second                                 |                                                              |
+| At-Least-Once                      | ◯             | ◯                                                            | ◯                                                            |
+| SLA(%)                             | >=99.9        | >=99.9                                                       | >=99.9                                                       |
+
```
Comment (masaaania) [inline docs/rfcs/0039-commonalize-cloud-service-interface.md:140]: 👏

File: docs/rfcs/0039-commonalize-cloud-service-interface.md [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+      bucket: {YOUR_BUCKET_NAME}
+      # Must be a service account with "Storage Object Admin (roles/storage.objectAdmin)" role on the given bucket
+      # since PipeCD need to write file object such as deployment log file to that bucket.
+      credentialsFile: /etc/pipecd-secret/gcs-service-account
+```
+
+https://pipecd.dev/docs/installation/install-controlplane/#using-firestore-and-gcs
+
+They use Helm and above YAML file.
+
```
Comment (masaaania) [inline docs/rfcs/0039-commonalize-cloud-service-interface.md:84]:
```suggestion

**GrowthBook**

```

**Review by masaaania — DISMISSED** [review]
LGTM!

File: docs/rfcs/0039-commonalize-cloud-service-interface.md [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+![data-pipeline](./images/0039-image4.png)
+
+We use Druid as a relay DB and usual DB(fething data directory) and Kafka as an intereface for Druid. The amount of data is a huge and we have to handle them as a high performance when fething data directory.
+
+#### Controversial topic
+
+##### 1. Can we stop using Druid & Kafka?
+
+We have the following problems when using Druid & Kafka.
```
Comment (cre8ivejp) [inline docs/rfcs/0039-commonalize-cloud-service-interface.md:211]: To support the multi-cloud platform, we will need to decide which DB will replace the Druid first, but we still don't know which DB will fit our usage. I would like to discuss this with you when you have time tomorrow.

**Review by masaaania — APPROVED** [review]
lgtm!

**Review by masaaania — APPROVED** [review]

**Pull request merged**